paket.dependencies

```diff
@@ -36,12 +36,25 @@ nuget Serilog.Sinks.Async
 nuget Destructurama.FSharp 1.1.1-dev-00035 # prerelease is stable, just has different FSharp.Core version constraints
 
 nuget Microsoft.SourceLink.GitHub 1.0.0-beta2-18618-05 copy_local:true, strategy:min
+nuget Microsoft.NETFramework.ReferenceAssemblies 1.0.0
 
 group Build
-  framework: net45
-  source https://api.nuget.org/v3/index.json
+  source https://www.nuget.org/api/v2
```

Comment:
I'd suggest moving this back to the v3 api, it's much much faster. also add `storage: none` and `frameworks: netstandard2.0` for least amount of overall processing time spent here.

Reply:
Done